app/decorators/middleware_deployment_decorator.rb

```diff
@@ -3,11 +3,11 @@ class MiddlewareDeploymentDecorator < Draper::Decorator
 
   def fonticon
     if name.end_with? '.ear'
-      'product-file-ear-o'
+      'product product-file-ear-o'
     elsif name.end_with? '.war'
-      'product-file-war-o'
+      'product product-file-war-o'
     else
-      'product-report'
+      'fa fa-file-text-o'
```

Comment:
@himdel Can you find the screen this displays on? I need to remove the "product" font class from that screen. Thanks.